woocommerce/woocommerce-ios
# Login: Add new screen for site credential login in the Jetpack setup flow

Part of #8075


### Description

This PR adds a new screen for signing in with site credentials for the Jetpack setup flow. Changes include:
- A new coordinator to handle the native Jetpack setup flow.
- A new view and view model for site credential login. 
- Updated the new Jetpack error screen with the entry point to the native Jetpack setup flow.
- Handled credential login.

### Testing instructions


Prerequisite: Make sure that you have a test store that is not associated with your WP.com account or doesn't have Jetpack.

- Log out of the app or skip login onboarding if needed.
- On the prologue screen, select "Log In" or "Continue with WordPress.com" based on the A/B test variant you get.
- Log in with your WordPress.com account.
- If your account doesn't have any associated site, notice that the empty store picker is displayed with the matching design as on Figma.
- Select Add a Store > Connect an existing site.
- Enter the address of your test store and tap Continue.
- On the Jetpack error screen, select Install Jetpack or Connect Jetpack.
- Notice that a new screen for site credential login is presented. The UI should match the design in ERGtCFLyiC9CFpHsw1XIid-fi-1110%3A37319. 
- Enter incorrect credentials, and notice that an alert is displayed with the appropriate message.
- Try again with the correct credentials, notice in Xcode console: `🎉`. The actual implementation will be handled in a subsequent PR.

### Screenshots

<img src="https://user-images.githubusercontent.com/5533851/202393271-ebab2f44-bbe6-4fa9-81af-e129d7236784.png" width=320 />

---
